com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.Debug.cs

```diff
@@ -30,6 +30,13 @@ public partial class HDRenderPipeline
 
         DebugDisplaySettings m_DebugDisplaySettings = new DebugDisplaySettings();
 
+        public static bool allowDebugMaterialSamplerOverride = UnityEngine.SystemInfo.hasMaterialSamplerOverride;
+        private static MaterialSamplerOverride s_MaterialDebugSamplerOverride = new MaterialSamplerOverride() { mipBias = 0.0f };
```

Comment:
I am not sure why we have a different workflow than the other debug Materials above?

Reply:
yup, it should be in Material section, not in lighting

Reply:
Will move this to material panel

Reply:
This has been addressed